cluster/extensions-javadsl/src/main/scala/com/lightbend/lagom/internal/javadsl/cluster/projections/ProjectorRegistryModule.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.lightbend.lagom.internal.javadsl.cluster.projections
+
+import java.util.concurrent.CompletionStage
+
+import com.lightbend.lagom.internal.cluster.projections.ProjectorRegistry
+import com.lightbend.lagom.javadsl.cluster.projections.DesiredStatus
+import com.lightbend.lagom.javadsl.cluster.projections.Projections
+import javax.inject.Inject
+import play.api.Configuration
+import play.api.Environment
+import play.api.inject.Binding
+import play.api.inject.Module
+
+import scala.compat.java8.FutureConverters
+import scala.concurrent.ExecutionContext
+
+class ProjectorRegistryModule extends Module {
```

Comment:
@ApiMayChange

Reply:
Hmmm, I just realised this is package `internal` and I'm not sure if it should be `non-internal` _and_ `ApiMayChange` or remain `internal` and forget about `ApiMayChange`.

Reply:
After checking other modules, it must not be `interal` so it must be `ApiMayChange`.

Reply:
correct, we should only use `ApiMayChange` for external stuff.